### ISSUE TITLE
Does not pull pfsense config

### ISSUE THREAD

**Issue by Dervish13** [ytti/oxidized#3109]
I recently upgraded our ancient oxidized to the latest release.
I have found that pfsense only contains comment of the release.

I have zero ruby knowledge, but I was able to trace it back to this commit https://github.com/ytti/oxidized/commit/00c1394110c7527c76bec4a73efd30f5083a0eea

Reintroducing modified version of array conversion block restores functionality.
Specifically 
```
 cmd :all do |cfg|
    cfg.each_line.to_a[0..-1].join
  end
```
restores the config, and keeps the first xml line, while the version comment is at the bottom.

**Comment by robertcheramy** [comment]
This Issue has been closed in the master branch, and will be included in [release 0.30](https://github.com/ytti/oxidized/milestone/4).

**Comment by Dervish13** [comment]
Thank you, just side note, I do not see code changes in master branch.

**Comment by TheMonkeyBrain** [comment]
This fix is still not present in the code for v0.31.0 or is it another fix than what Dervish31 suggested?

**Comment by robertcheramy** [comment]
I have no Idea why I stated this has been fixed. I remember testing pfsense and opensense, but that's all. There is no relevant commit in the pfsense model. Repoening this issue.
Duplicates: #2850 #3325

**Comment by robertcheramy** [comment]
I thought I had fixed this in #2771 .

**Comment by robertcheramy** [comment]
I tested with opnsense but not pfsense.

**Comment by robertcheramy** [comment]
pfsense wants me to create an user account to get the iso 🙈
Can you provide a YAML Simulation file for your PFSense?
Instructions: https://github.com/ytti/oxidized/blob/master/docs/Issues.md#sumbit-a-yaml-simulation-file
Command:
```
./device2yaml.rb user@host -c cmdsets/pfsense -e -o yaml/pfsense.yaml
```

**Comment by TheMonkeyBrain** [comment]
> pfsense wants me to create an user account to get the iso 🙈 Can you provide a YAML Simulation file for your PFSense? Instructions: https://github.com/ytti/oxidized/blob/master/docs/Issues.md#sumbit-a-yaml-simulation-file Command:
> 
> ```
> ./device2yaml.rb user@host -c cmdsets/pfsense -e -o yaml/pfsense.yaml
> ```

Sorry for not getting back to you; I hope to have time for it this week.

**Comment by TheMonkeyBrain** [comment]
I had some time to look into this today and it looks to actually be working if I use the pfsense model and not my own model.

I guess the confusion from my site have been that I didn't see any fix for this specific issue or any changes done to the pfsense model and apparently I hadn't noticed #2982 released with 0.30.

I will still create a [YAML simulation file](https://github.com/ytti/oxidized/blob/master/examples/device-simulation) and probably also a [model unit tests](https://github.com/ytti/oxidized/blob/master/spec/model)

**Comment by robertcheramy** [comment]
PR #3348 = unit test
